core/src/main/java/org/apache/accumulo/core/conf/ConfigurationDocGen.java

```diff
@@ -117,7 +117,13 @@ void property(Property prop) {
         // deal with multi-line values, skip strikethrough of value
         defaultValue = strike("**default value:** ", depr) + "\n```\n" + defaultValue + "\n```\n";
       } else {
-        defaultValue = strike("**default value:** " + "`" + defaultValue + "`", depr);
+        if (prop.getType() == PropertyType.CLASSNAME
+            && defaultValue.startsWith("org.apache.accumulo.core")) {
```

Comment:
I think this should be `startsWith("org.apache.accumulo")`.  We can update the jlink tag to handle different package prefeixes.